libs/dh/shared/domain/src/lib/generated/graphql.ts

```diff
@@ -371,6 +384,23 @@ export const mockGetPermissionLogsQuery = (resolver: ResponseResolver<GraphQLReq
     resolver
   )
 
+/**
+ * @param resolver a function that accepts a captured request and may return a mocked response.
+ * @see https://mswjs.io/docs/basics/response-resolver
+ * @example
+ * mockGetPermissionDetailsQuery((req, res, ctx) => {
+ *   const { id } = req.variables;
+ *   return res(
+ *     ctx.data({ permission })
+ *   )
+ * })
+ */
+export const mockGetPermissionDetailsQuery = (resolver: ResponseResolver<GraphQLRequest<GetPermissionDetailsQueryVariables>, GraphQLContext<GetPermissionDetailsQuery>, any>) =>
+  graphql.query<GetPermissionDetailsQuery, GetPermissionDetailsQueryVariables>(
+    'GetPermissionDetails',
+    resolver
+  )
```

Comment:
suggestion: consider mocking this query.

Reply:
https://github.com/Energinet-DataHub/greenforce-frontend/pull/1583/commits/5a7bb00b0549caf338840b6d77efb2015897a6d7